11.hello-dapr-slim/README.md

```diff
@@ -0,0 +1,148 @@
+# Hello World(without docker)
+
+This tutorial will demonstrate how to get Dapr running locally on your machine without docker. We'll be deploying a Node.js app that subscribes to order messages demostrating service invocation capability of Dapr. The following architecture diagram illustrates the components that make up the first part sample: 
+
+![Architecture Diagram](./img/Architecture_DiagramA.png)
+
+## Prerequisites
+This sample requires you to have the following installed on your machine:
+- [Node.js version 8 or greater](https://nodejs.org/en/) 
+- [Postman](https://www.getpostman.com/) [Optional]
+
+## Step 1 - Setup Dapr 
+
+Follow [instructions](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#environment-setup) to download and install the Dapr CLI and initialize Dapr.
+
+## Step 2 - Understand the Code
+
+Now that we've locally set up Dapr, clone the repo, then navigate to the Hello World sample: 
+
+```bash
+git clone https://github.com/dapr/samples.git
+cd samples/11.hello-dapr-slim
+```
+
+
+In the `app.js` you'll find a simple `express` application, which exposes a single route and handler. First, let's take a look at the top of the file: 
+
+Let's take a look at the ```neworder``` handler:
+
+```js
+app.post('/neworder', bodyParser.json(), (req, res) => {
+    const data = req.body.data;
+    const orderId = data.orderId;
+    console.log("Got a new order! Order ID: " + orderId);
+    res.status(200).send("Got a new order! Oredr ID: " + orderId);
+});
+```
+
+Here we're exposing an endpoint that will receive and handle `neworder` messages. We log the incoming message, and respond with the same log.
+
+## Step 3 - Run the Node.js App with Dapr
+
+1. Install dependencies: 
+
+    ```sh
+    npm install
+    ```
+
+    This will install `express` and `body-parser`, dependencies that are shown in our `package.json`.
+
+2. Run Node.js app with Dapr: 
+
+    ```sh
+    dapr run --app-id nodeapp --app-port 3000 --port 3500 node app.js
+    ```
+
+The command should output text that looks like the following, along with logs:
+
+```
+Starting Dapr with id nodeapp. HTTP Port: 3500. gRPC Port: 9165
+You're up and running! Both Dapr and your app logs will appear here.
+...
+```
+> **Note**: the `--app-port` (the port the app runs on) is configurable. Our Node app happens to run on port 3000, but we could configure it to run on any other port. Also note that the Dapr `--port` parameter is optional, and if not supplied, a random available port is used.
+
+The `dapr run` command looks for the default components directory which for Linux/MacOS is `$HOME/.dapr/components` and for Windows is `%USERPROFILE%\.dapr\components`. On initializing in self hosted mode without docker, the components folder is empty. 
```

Comment:
Should this paragraph be omitted?  They wouldn't be using any component in slim mode.